### DISCUSSION TITLE
How are orgs added?

### DISCUSSION THREAD

**Discussion by kungfuchicken** [jakecoffman/stldevs#83] [Q&A]
Hi @jakecoffman, how are orgs added to the site? I'm assuming it's data driven, but i don't see an endpoint in the API.

**Comment by jakecoffman** [comment]
Just as with users, orgs that identify as being in St. Louis will be included. Unfortunately the GitHub UI has been changed at some point so you have to use a workaround to set it: https://stackoverflow.com/questions/68970902/custom-location-text-for-github-organization

**Reply by kungfuchicken** [reply]
Got it. That makes sense. GH itself is the DB. 

Unfortunately Even the first answer there doesn't appear to work anymore. I'll poke at the second one using the API later. 

It seems like it might be worth thinking up a new way of tracking this, since most orgs are unlikely to do it this way anymore. I also noticed several orgs I have familiarity with are effectively defunct, so having a way of annotating with a bit of metadata might be useful. I don't know if that makes sense in terms of your goals, or if you would prefer for any annotation to be editorial (by you i would presume), crowdsourced, or somewhere in between.

**Reply by jakecoffman** [reply]
I migrated the orgs to a flat listing since it seems new orgs won't be able to set St. Louis as their location: https://github.com/stldevs/stldevs/blob/master/aggregator/orgs.txt

Feel free to put up a PR to add new orgs you're aware of.
